src/types/WidgetProps.ts

```diff
@@ -15,8 +15,15 @@ export type WidgetProps = {
     limit?: number;
     noCount?: boolean;
     noFilterByDailyEvents?: boolean;
+    params?: {
+      query: string;
+      queryVariables: {};
+      title: string;
+    };
+    routeName?: string;
     staticContentName?: string;
     staticContentTitle?: string;
+    title: string;
```

Comment:
This widget is made to work when you copy and paste an object from homeService. So I wanted it to be the same as the structure in homeService.

Reply:
Then I'll add two options. This widget can make both options available.

Reply:
what do you mean with two options? i spoke about removing one of them to have just one option. how do you see a benefit in having two options for the same purpose?

Reply:
If we remove text from other widgets then we will have to update widgets for all apps of all municipalities when we create a new version. But if we add the `text` option to `ServiceWidget`, then in the future we can add the widget name as `additionalProps.title` or `text` when creating widgets.

Reply:
i understand, then we should follow the logic: widget name visible = text and not have a second option `additionalProps.title`. it would be confusing to have it differently configured across all widgets.
i understand the idea to have the same props as for service tiles, but this makes it  unclear for widgets.
we do not need a 100% match in widgets to service tile.
the service tiles configs could be a point that needs refactoring.

Reply:
Okay. I agree with you, if we want to create a widget then we have to follow the rules! I remove the `additionalProps.title` property and add the `text` like all widgets.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/a8c3ad3d259c1c6064558e9871060429c9f884bf